libcodechecker/server/api/report_server.py

```diff
@@ -2111,7 +2116,8 @@ def massStoreRun(self, name, tag, version, b64zip, force):
                                          source_root,
                                          run_id,
                                          file_path_to_id,
-                                         run_history_time)
+                                         run_history_time,
+                                         severity_map)
```

Comment:
I think here you can use `context.severity_map`, so `severity_map` variable can be removed.

Reply:
done